duckduckgo/Android
# Add Scorecard

Asana Issue URL: https://app.asana.com/0/488551667048375/495411750237128

## Description
• Adds a scorecard to the app
• Pull out ui components from DashboardViewModel and moves them into stand alone renderers

## Steps to Test this PR:
TEST ONE:
1. Visit duckduckgo.com
1. Tap on the privacy grade
1. See that the scorecard is all good

TEST TWO:
1. Visit cnn.com
1. Tap on the privacy grade
1. See that the scorecard is all bad but DOES NOT display "Site is a major tracker network" displayed

TEST THREE:
1. Visit facebook.com
1. Tap on the privacy grade
1. See that the scorecard displays "Site is a major tracker network" displayed
